P4/dbCreateQuery.sql

```diff
@@ -0,0 +1,65 @@
+USE master
+GO
+
+CREATE DATABASE CommercialNetwork
+GO
+
+USE CommercialNetwork
+GO
+
+/*Product table*/
+CREATE TABLE [dbo].[Product](
+	[ProductID][int]IDENTITY(1,1) NOT NULL,
+	[ProductName][nvarchar](50) NOT NULL,
+	[Price][money],
+	PRIMARY KEY CLUSTERED ([ProductID])
+)
+GO
+
+/*Shop table*/
+CREATE TABLE [dbo].[Shop](
+	[ShopID] [int] IDENTITY(1,1) NOT NULL,
+	[ShopName] [nvarchar](100) NOT NULL,
+	[Address] [nvarchar](100) NOT NULL,
+	PRIMARY KEY CLUSTERED ([ShopID])
+)
+GO
+
+/*Order table*/
+CREATE TABLE [dbo].[Order](
```

Comment:
Seems done strictly by task, but actually good point will be to allow several products in one order at least for one shop - no need to redo, but keep in mind.